core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -0,0 +1,351 @@
+        'An svg diagram.');
+    }, true);
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    var explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    await explorationEditorPage.navigateToSettingsTab();
+    let width = (await browser.getWindowSize()).width;
+    if (width < 769) {
+      var basicSettings = $('.e2e-test-settings-container');
+      await action.click('Basic Settings', basicSettings);
+    }
+    await explorationEditorSettingsTab.setTitle('exp1');
+    await explorationEditorSettingsTab.setCategory('Algebra');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Dummy exploration for testing images'
+    );
+    await explorationEditorPage.saveChanges('Done!');
+    await workflow.publishExploration();
+    let dummyExplorationId = await general.getExplorationIdFromEditor();
+
+    // Adding the exploration to a curated lesson.
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.waitForTopicsToLoad();
+    await topicsAndSkillsDashboardPage.navigateToTopicWithIndex(0);
+    await topicEditorPage.createStory(
+      'Story Title', 'topicandstoryeditorone', 'Story description',
+      Constants.TEST_SVG_PATH);
+    await storyEditorPage.createNewChapter(
+      'Chapter 1', dummyExplorationId, Constants.TEST_SVG_PATH);
+    await storyEditorPage.updateMetaTagContent('story meta tag');
+    await storyEditorPage.saveStory('Saving Story');
+    await storyEditorPage.publishStory();
+
+    // Testing the copy tool.
+    let opportunityActionButtonCss = $(
+      '.e2e-test-opportunity-list-item-button');
+    let copyButton = $('.e2e-test-copy-button');
+    let doneButton = $('.e2e-test-close-rich-text-component-editor');
+    let saveButton = $('.e2e-test-save-button');
+    let textbox = $('.e2e-test-description-box');
+
+    await contributorDashboardPage.get();
+    await waitFor.pageToFullyLoad();
+    await contributorDashboardPage.navigateToTranslateTextTab();
+    await contributorDashboardTranslateTextTab.changeLanguage(
+      'shqip (Albanian)');
+    await contributorDashboardPage.waitForOpportunitiesToLoad();
+    await action.click('Opportunity button', opportunityActionButtonCss);
+    let image = $('.e2e-test-image');
+    await waitFor.visibilityOf(
+      image,
+      'Test image taking too long to appear.');
+    let images = await $$('.e2e-test-image');
+    expect(images.length).toEqual(1);
+
+    // Copy tool should copy image on pressing 'Done'.
+    await waitFor.visibilityOf(
+      copyButton, 'Copy button taking too long to appear');
+    await action.click('Copy button', copyButton);
+    await action.click('Image', images[0]);
+    await action.setValue(
+      'Set Image description', textbox, 'An example description');
+    await action.click('Done', doneButton);
+    await action.click('Save', saveButton);
+    await users.logout();
```

Comment:
Maybe  define this in utils file as  copyElementWithClassName(elementDescription, elementClassName) and use it here*

Reply:
Done. PTAL.